generative/maisi/large_files.yml

```diff
@@ -0,0 +1,23 @@
+large_files:
+  - path: "models/autoencoder_epoch273.pt"
+    url: "https://drive.google.com/file/d/1jQefG0yJPzSvTG5rIJVHNqDReBTvVmZ0/view?usp=drive_link"
+  - path: "models/input_unet3d_data-all_steps1000size512ddpm_random_current_inputx_v1.pt"
+    url: "https://drive.google.com/file/d/1FtOHBGUF5dLZNHtiuhf5EH448EQGGs-_/view?usp=sharing"
+  - path: "models/controlnet-20datasets-e20wl100fold0bc_noi_dia_fsize_current.pt"
+    url: "https://drive.google.com/file/d/1izr52Whkk56OevNTk2QzI86eJV9TTaLk/view?usp=sharing"
+  - path: "models/mask_generation_autoencoder.pt"
+    url: "https://drive.google.com/file/d/1FzWrpv6ornYUaPiAWGOOxhRx2P9Wnynm/view?usp=drive_link"
+  - path: "models/mask_generation_diffusion_unet.pt"
+    url: "https://drive.google.com/file/d/11SA9RUZ6XmCOJr5v6w6UW1kDzr6hlymw/view?usp=drive_link"
+  - path: "configs/candidate_masks_flexible_size_and_spacing_3000.json"
+    url: "https://drive.google.com/file/d/1yMkH-lrAsn2YUGoTuVKNMpicziUmU-1J/view?usp=sharing"
+  - path: "configs/all_anatomy_size_condtions.json"
+    url: "https://drive.google.com/file/d/1AJyt1DSoUd2x2AOQOgM7IxeSyo4MXNX0/view?usp=sharing"
+  - path: "datasets/all_masks_flexible_size_and_spacing_3000.zip"
+    url: "https://drive.google.com/file/d/16MKsDKkHvDyF2lEir4dzlxwex_GHStUf/view?usp=sharing"
+  - path: "datasets/IntegrationTest-AbdomenCT.nii.gz"
```

Comment:
Hi @yiheng-wang-nv , do you know if this file is for integration test only? I cannot find where this is used.